src/Runner.Common/Runner.Common.csproj

```diff
@@ -5,7 +5,6 @@
     <OutputType>Library</OutputType>
     <RuntimeIdentifiers>win-x64;win-x86;linux-x64;linux-arm64;linux-arm;osx-x64</RuntimeIdentifiers>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
-    <AssetTargetFallback>portable-net45+win8</AssetTargetFallback>
```

Comment:
this is a legacy setting we had a long time ago, seems fine to delete it now.

Reply:
We're no longer using the package that needed it 👍